Adjust base settings and add production settings

simonprast opened the pull request:
**Checklist:**

 <!--- Make sure your PR is documented and tested before submission. Put an `x` in all the boxes that apply: -->
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?


**Type of Changes:**

 <!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)


**What is the current behavior? (link to any open issues here)**
Production settings do not exist. #6 


**What is the new behavior (if this is a feature change)?**
I added documentation to the base.py settings file.
Additionally, I added Jinja2 as a templating language in the Base settings file. Jinja2 is more programmer-friendly than Django Template Language.
The production.py settings file is taken and adjusted from the Pharmaziegasse Website and therefore inspired by Florian Kleber.

Aichnerc commented on charm/home/models.py:
```diff
@@ -1,47 +1,116 @@
+#xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
 from django.db import models
-from wagtail.admin.edit_handlers import FieldPanel, TabbedInterface, ObjectList, PageChooserPanel
+from wagtail.admin.edit_handlers import FieldPanel, StreamFieldPanel, MultiFieldPanel, TabbedInterface, ObjectList, PageChooserPanel
```
The max char count per line is 119.
How to fix: https://www.python.org/dev/peps/pep-0328/

simonprast committed:
Adjust base settings and add production settings

I added documentation to the base.py settings file.
Additionally, I added Jinja2 as a templating language in the Base
settings file. Jinja2 is more programmer-friendly than Django Template
Language.
The production.py settings file is taken and adjusted from the
Pharmaziegasse Website and therefore inspired by Florian Kleber.

Aichnerc reviewed (approved):
Looks fine to me.
